webcompat/views.py

```diff
@@ -179,14 +179,7 @@ def create_issue():
             flash(msg, 'notimeout')
             return redirect(url_for('index'))
     form['ua_header'] = request.headers.get('User-Agent')
-    # Currently we support either a src GET param, or X-Reported-With header
-    # to track where the report originated from.
-    # See https://github.com/webcompat/webcompat.com/issues/1254 to track
-    # supporting only the src param
-    if session.get('src'):
-        form['reported_with'] = session.pop('src')
-    else:
-        form['reported_with'] = request.headers.get('X-Reported-With', 'web')
+    form['reported_with'] = session.pop('src')
```

Comment:
So `src` will end up in the `session` if there was a `src` param:

https://github.com/webcompat/webcompat.com/blob/a5b5d84a06cbf1564247208ac8bf8e825134326d/webcompat/views.py#L166

(for all add-on and "report site issue "reported issues, that gets added for the user).

If this was a web-reported issue, `src` won't be in the session and a `KeyError` will be raised (which means the page will explode, as is).

[`dict#pop`](https://docs.python.org/2/library/stdtypes.html#dict.pop) does allow you to specify a default value if the key isn't there though, as the second arg. 

So I think we want `session.pop('src', 'web')` -- which should mean if there wasn't a `src` param added by an add-on, we should set `reported_with` to `web`.

Does that make sense, @zoepage? Anything I'm overlooking?